pkg/process/net/types_unsupported.go

```diff
@@ -1,4 +1,4 @@
-// +build !linux
+// +build !linux,!windows
```

Comment:
nit: you need to update the comments in this file since they're all referencing linux specifically